### ISSUE TITLE
Only show Codelens for certain test files

### ISSUE THREAD

**Issue by ThomasStock** [90poe/vscode-cy-helper#60]
In our project we have both 'normal' unit tests and cypress tests.
With the extension, we get the `open cypress | Set "only"` codelens options for unit tests as well.

It would be great if we could limit the codelens to a certain folder or filepath filter instead of showing for every `it()`.

**Comment by Shelex** [comment]
Hi @ThomasStock 
Thank you for opening an issue.
Due to publishing token expiration and missing option to regenerate new one I have published fix for this issue and plugin itself under my personal account from fork.

 - [Cypress Helper v2](https://marketplace.visualstudio.com/items?itemName=shevtsov.vscode-cy-helper)
 - [Repository](https://github.com/Shelex/vscode-cy-helper)
 - In case you have found this extension via pack, it is also republished - [extension pack](https://marketplace.visualstudio.com/items?itemName=shevtsov.qa-extension-pack)
 
 Basically it is solved by providing a configuration option `cypressHelper.cypressCodeLensePattern` where you can specify glob pattern for folders which should enable code lenses.
 By default it will check files only inside `cypress` folder with extensions `.js`, `.ts` and `.feature`.
 You can customize default pattern by specifying your folder, for example if you have integration/e2e tests inside folder `e2e`, then you have to change default pattern to:
 `"cypressHelper.cypressCodeLensePattern": "**/e2e/**/*.+(js|ts|feature)"`.
